style.css

```diff
@@ -537,6 +598,132 @@ body {
     border: 0.25rem solid #0d151e;
     border-radius: 1rem;
   }
+
+  .project-details-modale {
+    display: flex;
+    flex-direction: column;
+    background-color: #fff;
+    border-radius: 1.6rem;
+    padding: 1.6rem;
+  }
+
+  .project-details-modale h3 {
+    font-size: 3.2rem;
+    font-weight: 700;
+    line-height: 4.4rem;
+    color: #172b4d;
+  }
+
+  .project-details-modale .project-img {
+    width: 100%;
+    height: auto;
+  }
+
+  .project-details-modale .description {
+    color: #344563;
+    font-size: 1.5rem;
+    font-weight: 400;
+    line-height: 2.4rem;
+    margin-bottom: 1.6rem;
+  }
+
+  .project-details-modale .project-highlights-modal {
+    margin: 0;
+    justify-self: flex-end;
+  }
+
+  .project-highlights-modal ul li:nth-child(2)::before {
+    content: url("./assets/images/Counter.png");
+    margin: 0 1.2rem;
+  }
+
+  .project-highlights-modal ul li:nth-child(3)::before {
+    content: url("./assets/images/Counter.png");
+    margin: 0 1.2rem;
+  }
+
+  .project-details-modale .btnActions {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    padding: 0 1rem;
+    row-gap: 1rem;
+  }
```

Comment:
- [ ] Ouch! this is close but please make sure you stick to the Figma design when implementing the layout of different elements as I noticed the buttons on the popup card are aligned vertically instead of horizontally as shown in the Figma design.
- Figma design:
  ![firefox_BTEgtFTGd2](https://github.com/AjroudiRida/Portfolio/assets/104968236/d370eb9a-0d33-4481-8c7a-4fcffb9a7d39)
- Your design:
  ![firefox_vDj8guBJqo](https://github.com/AjroudiRida/Portfolio/assets/104968236/91569900-5357-4497-8cde-049a12907b56)